tvbwidgets/core/pse/parameters.py

```diff
@@ -245,6 +247,16 @@ def _init_checkpoint(self):
     def monitor_execution(self):
         if self.update_progress is not None:
             self.update_progress()
+        else:
+            with self.progress_file_lock:
```

Comment:
Can you please benchmark if this addition is slowing down the PSE a lot ?
For example by locally timing "python -m tvb...parameters ...." with at least 5 x 5 simulations

Reply:
I found that there is no difference in the execution time between counting the finished job in a file (by locking that file at every write) and not counting it at all.

Here are the benchmarks:
with file writing:
-   Elapsed time: 318.0762997
-   Elapsed time: 312.1561808
-   Elapsed time: 314.2436308
-   Elapsed time: 311.7296136

without:
-   Elapsed time: 315.0625516
-   Elapsed time: 312.2725683
-   Elapsed time: 314.6274175
-   Elapsed time: 317.1361311